scripts/python/qa-scripts/createVCAAggregate.py

```diff
@@ -0,0 +1,523 @@
+from gspreadWrapper import GspreadWrapper
+from options import Options
+from utils import Utils
+
+from gspread.models import Cell
+from gspread_formatting import *
+
+from time import sleep
+import pandas as pd
+import json
+import os
+
+class createVCAAggregate():
+    def __init__(self):
+        self.opt = Options()
+        self.utils = Utils()
+        self.gspreadWrapper = GspreadWrapper()
+        self.proposals = json.load(open('proposals.json'))
+        self.vcas = json.load(open('vcas.json'))
+        self.vcasFiles = []
+
+        self.badColumns = [self.opt.notValidCol]
+        self.goodColumns = [self.opt.goodCol]
+        self.excellentColumns = [self.opt.excellentCol]
+        self.allColumns = self.badColumns + self.goodColumns + self.excellentColumns
+
+        self.dChallenges = self.opt.distinctChallenges
+
+
+    def prepareBaseData(self):
+        self.gspreadWrapper.getVCAMasterData()
+        self.vcaMerged = pd.DataFrame(columns=self.gspreadWrapper.dfVca.columns, data=None)
+        self.vcaMerged['name'] = ''
+        self.dfVca = self.gspreadWrapper.dfVca.set_index('id')
```

Comment:
same as before, would suggest to use the appropriate opt header instead of a literal string